addons/website_slides/static/src/js/slides_course_tag_add.js

```diff
@@ -0,0 +1,328 @@
+odoo.define('website_slides.channel_tag.add', function (require) {
+'use strict';
+
+var core = require('web.core');
+var Dialog = require('web.Dialog');
+var publicWidget = require('web.public.widget');
+
+var _t = core._t;
+
+var TagCourseDialog = Dialog.extend({
+    template: 'website.slides.tag.add',
+    events: _.extend({}, Dialog.prototype.events, {
+        'change input#tag_id' : '_onChangeTag',
+    }),
+
+    /**
+    * @override
+    * @param {Object} parent
+    * @param {Object} options holding channelId
+    *      
+    */
+    init: function (parent, options) {
+        options = _.defaults(options || {}, {
+            title: _t("Add a tag"),
+            size: 'medium',
+            buttons: [{
+                text: _t("Add"),
+                classes: 'btn-primary',
+                click: this._onClickFormSubmit.bind(this)
+            }, {
+                text: _t("Discard"),
+                close: true
+            }]
+        });
+
+        this.channelID = parseInt(options.channelId, 10);
+        if (options.channelTagIds) {
+            this.tagIds = options.channelTagIds;
```

Comment:
this.tagIds = options.channelTagIds || []; ?

Reply:
Done